test/Qowaiv.DomainModel.UnitTests/Commands/CommandProcessor_specs.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using Qowaiv.DomainModel.Commands;
+using Qowaiv.Validation.Abstractions;
+using System;
+using System.Threading.Tasks;
+
+namespace Commands.CommandProcessor_specs
+{
+    public class Sends
+    {
+        [Test]
+        public async Task a_command()
+        {
+            var processor = new NumberProcessor(new NumberHandler());
+            var response = await processor.Send(new One());
+            response.Value.Should().Be(1);
+        }
+
+        [Test]
+        public async Task a_second_command_using_precompiled_func()
```

Comment:
Whats the difference between the first and the second test? They both are registered in the same way.

Reply:
It is hard to proof that the second call follows the per-compiled path. With this test, code coverage detects this line. But from test perspective, that is not a proof, obviously.